CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log - AWS AppSync SDK for Android
 
+## [Release 2.6.27](https://github.com/awslabs/aws-mobile-appsync-sdk-android/releases/tag/release_v2.6.27)
+
+### Bug Fixes
+* Fixed bug in subscribe call that was setting the QOS to be 0. It is now correctly set to QOS 1. See [issue #54](https://github.com/awslabs/aws-mobile-appsync-sdk-android/issues/54)
+* Fixed bug that was causing a ConcurrentModificationException in the logic to handle connection loss. See [PR #22](https://github.com/awslabs/aws-mobile-appsync-sdk-android/pull/22) and [issue #53](https://github.com/awslabs/aws-mobile-appsync-sdk-android/issues/53)
```

Comment:
```suggestion
* Fixed bug that was causing a ConcurrentModificationException in the logic to handle connection loss. See [issue #22](https://github.com/awslabs/aws-mobile-appsync-sdk-android/issues/22) and [issue #53](https://github.com/awslabs/aws-mobile-appsync-sdk-android/issues/53)
```

Reply:
Done. It was PR 41 for issue 22 - I have changed it to the right PR#.